.build/azure-pipeline-nuget.yml

```diff
@@ -1,260 +1,247 @@
-    condition: eq('${{parameters.doRelease}}', true)
-
-    steps:
-    - checkout: none
-    - task: DownloadBuildArtifacts@1
-      displayName: Download the NuGet Packages from Build System
-      inputs:
-        buildType: 'current'
-        downloadType: 'specific'
-        itemPattern: '**/*.NUnit.*'
-        downloadPath: '$(System.ArtifactsDirectory)'
-
-    - task: NuGetCommand@2
-      displayName: Push NUnit NuGet to Nuget.org
-      inputs:
-        command: 'push'
-        packagesToPush: '$(System.ArtifactsDirectory)/**/*.nupkg'
-        nuGetFeedType: 'external'
-        publishFeedCredentials: 'NuGet-Playwright'
-
-- stage: ReleasePlaywrightMSTest
-  dependsOn: BuildPackageSign
-
-  jobs:
-  - job: ReleaseNugetMSTest
-    displayName: Publish Playwright.MSTest on Nuget.org
-    condition: eq('${{parameters.doRelease}}', true)
-
-    steps:
-    - checkout: none
-    - task: DownloadBuildArtifacts@1
-      displayName: Download the NuGet Packages from Build System
-      inputs:
-        buildType: 'current'
-        downloadType: 'specific'
-        itemPattern: '**/*.MSTest.*'
-        downloadPath: '$(System.ArtifactsDirectory)'
-
-    - task: NuGetCommand@2
-      displayName: Push MSTest NuGet to Nuget.org
-      inputs:
-        command: 'push'
-        packagesToPush: '$(System.ArtifactsDirectory)/**/*.nupkg'
-        nuGetFeedType: 'external'
-        publishFeedCredentials: 'NuGet-Playwright'
-
-- stage: ReleasePlaywrightTestAdapter
-  dependsOn: BuildPackageSign
-
-  jobs:
-  - job: ReleaseNugetTestAdapter
-    displayName: Publish Playwright.TestAdapter on Nuget.org
-    condition: eq('${{parameters.doRelease}}', true)
-
-    steps:
-    - checkout: none
-    - task: DownloadBuildArtifacts@1
-      displayName: Download the NuGet Packages from Build System
-      inputs:
-        buildType: 'current'
-        downloadType: 'specific'
-        itemPattern: '**/*.TestAdapter.*'
-        downloadPath: '$(System.ArtifactsDirectory)'
-
-    - task: NuGetCommand@2
-      displayName: Push TestAdapter NuGet to Nuget.org
-      inputs:
-        command: 'push'
-        packagesToPush: '$(System.ArtifactsDirectory)/**/*.nupkg'
-        nuGetFeedType: 'external'
-        publishFeedCredentials: 'NuGet-Playwright'
+- name: doRelease
+  displayName: Push the Playwright Release to NuGet.org
+  default: false
+  type: boolean
+
+- name: doReleaseCLI
+  displayName: Push the CLI Release to NuGet.org
+  default: false
+  type: boolean
+
+- name: signType
+  displayName: Sign Type
+  default: 'test'
+  type: string
+  values:
+  - test
+  - real
+
+variables:
+- name: BuildConfiguration
+  value: 'Release' 
+
+resources:
+  repositories:
+  - repository: 1esPipelines
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      name: VSEngSS-MicroBuild2022-1ES
```

Comment:
Note: This was previously defined in the AzDo web-interfaces, this moves it into the yaml.

Reply:
In the web-interface there was autocomplete. How do we find a new value once this one goes out of fashion? Let's add instructions/link here?

Reply:
It was previously hard-coded without auto-complete, a string variable. See https://aka.ms/MicroBuild for the future.